openedx/adg/admin_dashboard/applications/helpers.py

```diff
@@ -0,0 +1,95 @@
+"""
+Helpers for `admin_dashboard.applications` app.
+"""
+import json
+from datetime import datetime, timedelta
+
+from openedx.adg.lms.applications.models import UserApplication
+from openedx.adg.lms.applications.rules import is_adg_admin
+
+
+def get_submitted_applications(user):
+    """
+    Returns all applications that have been submitted, for ADG admin, and for business unit admin, it returns only those
+    submitted applications that have been applied against the business unit that the user is admin of.
+
+    Arguments:
+        user (User): admin for whom submitted applications are to be fetched
+
+    Returns:
+        QuerySet: queryset containing submitted applications
+    """
+    submitted_applications = UserApplication.objects.prefetch_related(
+        'user__application_hub__submission_date').filter(
+        user__application_hub__is_written_application_completed=True)
+
+    if not (user.is_superuser or is_adg_admin(user)):
+        submitted_applications = submitted_applications.filter(business_line__group=user.groups.first())
+
+    return submitted_applications
```

Comment:
How about instead of using UserApplication, we use the ApplicationHub?

Reply:
Actually we have to use business_line and status fields of UserApplication model and just the submission_date of the ApplicationHub model. So it makes sense to use UserApplication and just prefetch submission_date.